example/README.md

```diff
@@ -257,8 +259,8 @@ with the following parameters:
 - AZone - Availability zone, needs to match the Subnet ID's availability zone
 - Ngap - `true` if in an NASA NGAP environment, will add the NGAP permission boundary to the IAM role created
 
-In the outputs section of your Cloudformation deployment in the AWS console, you can find the IP address of the fake server created. To use this fake server with the tests, update the provider configurations in `example/data/providers` to use this host address.
```

Comment:
I don't think this is relevant any longer but please correct me if I am wrong.

Reply:
Yeah, this is no longer relevant. Good catch